tests/test_cli.py

```diff
@@ -124,6 +124,29 @@ def test_cli_geometry_alternatives(run_cli, geojson_string, wkt_string):
     run_cli("--geometry", wkt_string, "--end", "20200101", "--limit", "1")
 
 
+@pytest.mark.vcr
+@pytest.mark.scihub
+def test_cli_gnss(run_cli):
+    run_cli(
+        "--gnss",
+        "-s",
+        "20210201",
+        "-e",
+        "20210202",
+        "--producttype",
+        "AUX_POEORB",
+        "--query",
+        "platformserialidentifier=1B",
+    )
+
+
+@pytest.mark.vcr
+@pytest.mark.scihub
+def test_cli_geometry_alternatives(run_cli, geojson_string, wkt_string):
```

Comment:
Looks like you accidentally duplicated this test.

Reply:
ups, sorry
should be fixed now